inventory/group_vars/all.yml

```diff
@@ -10,6 +10,10 @@ local_release_dir: "/tmp/releases"
 # Random shifts for retrying failed ops like pushing/downloading
 retry_stagger: 5
 
+# Controll delayed startup/restart for services, in seconds
```

Comment:
Please move these out of group_vars. All vars need defaults in their corresponding role. This config is not interesting for most users at all. It just needs to work.

Secondly, "Control" is misspelled.